package.json

```diff
@@ -2,18 +2,19 @@
   "name": "yscope-log-viewer",
   "version": "0.1.0",
   "description": "A tool tailored for viewing logs, including log formats designed by YScope.",
-  "main": "src/index.tsx",
+  "main": "src/main.tsx",
   "scripts": {
-    "analyze:size": "webpack --config webpack.prod.js --profile --json=bundle-stats.json && webpack-bundle-analyzer bundle-stats.json dist/",
-    "build": "webpack --config webpack.prod.js",
+    "build": "vite build",
+    "dev": "vite",
     "lint": "npm run lint:check",
     "lint:check": "npm-run-all --sequential --continue-on-error lint:check:*",
     "lint:check:css": "stylelint src/**/*.css",
     "lint:check:js": "eslint . --max-warnings 0",
     "lint:fix": "npm-run-all --parallel --continue-on-error \"lint:check:* -- --fix\"",
-    "start": "webpack serve --open --config webpack.dev.js",
```

Comment:
Alternatively, we can keep `start` as an alias of `dev`. e.g.,
```
    "start": "npm run dev",
```
And this PR won't necessarily be breaking.

Reply:
Right, it only affects developers. I updated the title to remove the `!` accordingly.